python/python/benchmarks/test_search.py

```diff
@@ -168,3 +169,28 @@ def test_ivf_pq_index_search(test_dataset, benchmark):
         ),
     )
     assert result.num_rows > 0
+
+
+@pytest.mark.benchmark(group="query_ann")
+@pytest.mark.parametrize(
+    "selectivity, prefilter, use_index",
+    list(itertools.product((0.25, 0.75), (False, True), (False, True))),
+)
```

Comment:
I think if you do this, it's a little more readable and also pytest might generate easier to understand test names:
```suggestion
@pytest.mark.parametrize("selectivity", (0.25, 0.75))
@pytest.mark.parametrize("prefilter", (False, True))
@pytest.mark.parametrize("use_index", (False, True))
```